.gitignore

```diff
@@ -7,3 +7,4 @@ results/
 testing/
 testing*
 *.pyc
+*swp
```

Comment:
is there a reason you added this?

Reply:
I guess that was for my own sanity. Using vim as an editor creates these temporary files. I can remove it